SeBa fails to compile

**Describe the bug**
@risojevicj writes:

> I ran into problems with amuse-seba. In src/amuse_seba/toolbox.patch I had to change
> -CXXFLAGS  += -I./include -I./include/star -DHAVE_CONFIG_H -DTOOLBOX
> to
> -CXXFLAGS  += -I./include -I./include/star -DHAVE_CONFIG_H -DTOOLBOX -Wconversion.
> After that, the build process started, but failed with
> /home/vlado/anaconda3/envs/amuse/bin/../lib/gcc/x86_64-conda-linux-gnu/13.3.0/../../../../x86_64-conda- linux-gnu/bin/ld: cannot find src/node/libnode.a: No such file or directory collect2: error: ld returned 1 exit status make: *** [Makefile:53: seba_worker] Error 1
> Apparently, src/node/libnode.a library is missing.

This is on 2025.5.0.rc2.

**To Reproduce**
./setup install seba

**Expected behavior**
SeBa compiles and installs correctly.

**Logs**
If applicable, add logfiles to help explain your problem.

@risojevicj, could you attach `support/logs/install-amuse-seba.log`? 

**Environment (please complete the following information):**
 - OS and version: Linux I think
 - Compiler: gcc 13.3.0

--- a/src/amuse_seba/toolbox.patch
+++ b/src/amuse_seba/toolbox.patch
@@ -3,8 +3,8 @@ index 63009a5..f4f744f 100644
 --- a/Makefile
 +++ b/Makefile
 @@ -1,4 +1,4 @@
--CXXFLAGS  += -I./include -I./include/star -DHAVE_CONFIG_H -DTOOLBOX
-+CXXFLAGS  += -I./include -I./include/star -DHAVE_CONFIG_H
+-CXXFLAGS  += -I./include -I./include/star -DHAVE_CONFIG_H -DTOOLBOX -Wconversion
++CXXFLAGS  += -I./include -I./include/star -DHAVE_CONFIG_H -Wconversion
  LDLIBS	  += -Lsstar -lsstar -Lnode -lnode -Lstd -lstd -lm
  CFLAGS    += -O
  CXXFLAGS  += -O

